libs/zedUpload/datastore.go

```diff
@@ -149,6 +149,7 @@ func (ctx *DronaCtx) postChunk(req *DronaRequest, chunkDetail ChunkData) {
 //	make sure the reply is always sent back
 func (ctx *DronaCtx) postResponse(req *DronaRequest, status error) {
 	// status is already set up by action, we just have to set processed flag
+	defer func() { recover() }()
```

Comment:
Should't we log the (non-nil) result of recover?

Reply:
We could, but all we will get is "wrote to closed channel". The info about the cause of the error already gets logged in other goroutines

Reply:
Yetus complained, so I've updated it to capture and log the error